modules/core/src/main/java/org/locationtech/jts/geom/Coordinate.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Vivid Solutions.
+ * Copyright (c) 2018 Vivid Solutions, and others.
```

Comment:
how about `Copyright (c) 2018 Vivid Solutions, and contributors.`? That is what most projects do and you don't have the overhead of contacting every previous contributor and hope they sign a copyleft contract.

Reply:
I don't mind either way, "and others" was how the eclipse projects were doing it.

Really we can ask @jnh5y as a locaitontech committer representative to look this up for us and get back with an answer.

Reply:
From looking at https://www.eclipse.org/projects/handbook/#ip-copyright-headers, it looks like "and Contributors to the Eclipse Foundation" may be acceptable.  (I'd be comfortable with that since it is listed in the Eclipse documentation.)

Reply:
I am just reverting it, we can sort out headers another time without delaying this PR.